LeetCodeAssistant/Utility.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Utility.swift
+//  LeetCodeAssistant
+//
+//  Created by Ryo Togashi on 2019-09-22.
+//  Copyright © 2019 Kohei Asai. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    func replaceHtmlTag() -> String {
+        return replacingOccurrences(of: "<(￥\"[^\"]*\"|'[^']*'|[^'\">])*>", with: "", options: .regularExpression, range: self.range(of: self)).replacingOccurrences(of: "&quot;", with: "").replacingOccurrences(of: "&nbsp;", with: "").replacingOccurrences(of: "C&#39;s", with: "").replacingOccurrences(of: "&#39;", with: "")
```

Comment:
@axross Can you tell me if you have a better idea to delete HTML tag.
and I just realized that this doesn't work for HTML Tag which has a Breaking-Line.
ex:
< a href="......."
      class="...." >

Reply:
@ryotogashi How's insert detection for linefeed?